src/main/scala/HellaCachePrefetcher.scala

```diff
@@ -12,15 +12,15 @@ import freechips.rocketchip.diplomacy._
 
 object HellaCachePrefetchWrapperFactory {
   def apply(hartIds: Seq[Int], prefetcher: CanInstantiatePrefetcher, base: BaseTile => Parameters => HellaCache) = (tile: BaseTile) => (p: Parameters) => {
```

Comment:
Should the hartId -> tileId naming be changed here as well? (and in Configs.scala?)

Reply:
Good catch